pwvlt/src/vault.rs

```diff
@@ -3,18 +3,23 @@ use crate::keyring_store::KeyringStore;
 use crate::nitrokey_store::NitrokeyStore;
 use crate::pass_store::PassStore;
 
+macro_rules! push_store {
+    ($v: expr, $store: expr, $name: tt) => {
```

Comment:
I don't really think `name` is needed here (we can implement `Display` for `NitrokeyStore` and `KeyringStore`).

Reply:
Implementing `Display` is not going to work in this case. `$store` is an `Option<Nitrokey/KeyringStore>`, and I am trying to display a message in the `None` case. If I implement `Display` for the two storage types, I will end up with a compile error, since `Option<Store>` doesn't implement `Display`.

Reply:
Fair enough, I am happy to merge this then.